### PR TITLE
Fixed #270: Created a manual github action that releases skupper-router

### LINKED ISSUE #270: Create a manual github workflow for releasing skupper-router
Create a new manual workflow where you can select a git tag and release the skupper-router

### PR DESCRIPTION
**Pull request by ganeshmurthy**

### CODE REVIEW

**Comment by ganeshmurthy** [comment]
I found this "release" event - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
I tried using it in this yml thinking that when I add the following to the yml - 
```
on:
  release:
    types: [published]
```
The job would trigger when we published the release but that did not happen. Not sure what I was missing

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Release skupper-router
+#on:
+#  release:
+#    types: [published]
```
Comment (jiridanek) [inline .github/workflows/release.yml:23]: Not sure why this does not work. One alternative would be to do

```
on:
  push:
    tags:
      - 'v**'
```

to trigger this when version tag (starting with `v`) is pushed.

Comment (jiridanek) [reply .github/workflows/release.yml:23]: In any case, it might be best to leave this step manual. Just so that weird things don't happen on their own.

**Review by fgiorgetti — APPROVED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build and publish skupper-router image
+    runs-on: ubuntu-latest
+    steps:
```
Comment (fgiorgetti) [inline .github/workflows/release.yml:31]: @ganeshmurthy one thing that might be interesting to evaluate would be the
use of create-release action, see: [](https://github.com/actions/create-release)

This manual task works ok, from the POV of releasing images, but there won't be
any release related information on the repository.

Comment (ganeshmurthy) [reply .github/workflows/release.yml:31]: > @ganeshmurthy one thing that might be interesting to evaluate would be the use of create-release action, see: https://github.com/actions/create-release
> 
> This manual task works ok, from the POV of releasing images, but there won't be any release related information on the repository.

Yes, Please see my earlier comment in this PR. I tried using the on: release which did not work. @jiridanek suggested a fix to my approach but then we agreed to keep this process manual.

**Pull request merged**